packages/react/src/index.d.ts

```diff
@@ -3,1160 +3,1121 @@
+        requiredControl?: boolean;
+    }
+    export interface FormInputProps extends BasicWithAsProps {
+        type?: string;
+        value?: string | number;
+        id?: string;
+        disabled?: boolean;
+        required?: boolean;
+        readOnly?: boolean;
+        sizeInput?: InputSize;
+        isValid?: boolean;
+        isInvalid?: boolean;
+        isBorderNone?: boolean;
+        isBackgroundReset?: boolean;
+    }
+    export interface FormLabelProps extends BasicProps {
+        sizeLabel?: InputSize;
+        required?: boolean;
+        htmlFor?: string;
+    }
+    export interface FormSelectProps extends BasicProps {
+        value?: string | number;
+        id?: string;
+        disabled?: boolean;
+        required?: boolean;
+        sizeInput?: InputSize;
+        isValid?: boolean;
+        isInvalid?: boolean;
+        isBorderNone?: boolean;
+        isBackgroundReset?: boolean;
+    }
+    export const Form: React.FC<BasicProps> & {
+        Check: React.FC<FormCheckProps>;
+        Feedback: React.FC<FormFeedbackProps>;
+        File: React.FC<FormFileProps>;
+        Group: React.FC<FormGroupProps>;
+        Input: React.FC<FormInputProps>;
+        Label: React.FC<FormLabelProps>;
+        InputGroup: React.FC<BasicProps>;
+        Select: React.FC<FormSelectProps>;
+    };
+    export interface HeaderProps extends BasicWithAsProps {
+        show?: boolean;
+        innerClassName?: string;
+        fullWidth?: boolean;
+    }
+
+    export const Header: React.FC<HeaderProps> & {
+        Left: React.FC<BasicWithAsProps>;
+        AbsoluteCenter: React.FC<BasicWithAsProps>;
+        Right: React.FC<BasicWithAsProps>;
+        Main: React.FC<BasicWithAsProps>;
+        Brand: React.FC<BasicWithAsProps>;
+    };
+
+    export interface HeaderMobileProps extends BasicProps {
+        show?: boolean;
+        showMenu?: boolean;
+        hasDropContext?: boolean;
+        onToggle?: () => void;
+    }
+    export const HeaderMobile: React.FC<HeaderMobileProps> & {
+        Main: React.FC<BasicProps>;
+        Brand: React.FC<BasicProps>;
+        AfterContext: React.FC<BasicProps>;
+        Context: React.FC<BasicProps>;
+        DropContext: React.FC<BasicProps>;
+    };
+
+    export interface IconProps {
+        name?: IconType;
+        size?:
+            | 'tiny'
+            | 'extraSmall'
+            | 'small'
+            | 'medium'
+            | 'large'
+            | 'extraLarge'
+            | 'extraLargePlus'
+            | 'huge';
+
+        path?: string;
+        style?: React.CSSProperties;
+    }
+    export const Icon: React.FC<IconProps>;
+
+    export interface LoaderProps extends BasicProps {
+        size?: InputSize;
+        duration?: number;
+    }
+    export const Loader: React.FC<LoaderProps>;
+
+    export interface LogoProps extends BasicProps {
+        name?: string;
+        src?: string;
+        alt?: string;
+        width?: number;
+        height?: number;
```

Comment:
These two can receive string value too

Reply:
Please see implemented of Logo, width and height will be paste to img tag not style Ref: https://www.w3schools.com/tags/tag_img.asp